src/__init__.py

```diff
@@ -8941,18 +8941,18 @@ def get_textpage(self, clip: rect_like = None, flags: int = 0, matrix=None) -> "
         textpage = TextPage(textpage)
         return textpage
 
-    def get_texttrace(self):
+    def get_texttrace(self, names=None):
 
         CheckParent(self)
         old_rotation = self.rotation
         if old_rotation != 0:
             self.set_rotation(0)
         page = self.this
         rc = []
-        if g_use_extra:
-            dev = extra.JM_new_texttrace_device(rc)
+        if g_use_extra or names is True:
```

Comment:
I have a MuPDF commit waiting for review that adds a SWIG-friendly `fz_get_glyph_name2()`, same as `fz_get_glyph_name()` except that it returns a std::string, so can be used from Python/C#. This will allow the pure-Python path in this fn to be used.

Might be worth waiting until this commit is in MuPDF master before merging this PR.